.travis.yml

```diff
@@ -16,3 +18,13 @@ script:
 after_success:
   - coverage run --source inicheck -m pytest
   - coveralls
+
+stages:
+  - name: deploy
+    if:  branch = master AND tag IS present AND repo = USDA-ARS-NWRC/inicheck
```

Comment:
I don't think so. What are your thoughts?

Reply:
We can always make this an:
```
branch = master OR tag IS present AND repo = USDA-ARS-NWRC/inicheck
```
If no deploy gets triggered

Reply:
So that would always deploy if we had a push to master? I am not sure how differently scm manages the tags but wouldn't that create a push to the same tag on pypi if we didn't create a new tag?

Reply:
That is a Scott question

Reply:
So this is my understanding:
1. Open a new PR with a tagged commit for the release
2. Merge to master

https://docs.travis-ci.com/user/deployment/pypi/#deploying-tags

Looks like we can also get rid of the branch conditional, as it defaults to master only:
https://docs.travis-ci.com/user/deployment/pypi/#deploying-specific-branches

Reply:
https://blog.travis-ci.com/2013-10-02-continuous-deployment-pypi-npm-and-more/